python/src/equistore/data/extract.py

```diff
@@ -51,8 +49,11 @@ def register_external_data_wrapper(origin, klass):
     Register a non-Python data origin and the corresponding class wrapper.
 
     The wrapper class constructor must take two arguments (raw ``eqs_array`` and
-    python ``parent`` object) and return a subclass of either ``numpy.ndarray``
-    or ``torch.Tensor``, which keeps ``parent`` alive.
+    python ``parent`` object) and return a subclass of either
+    :py:class:`numpy.ndarray` or :py:class:`torch.Tensor`, which keeps
+    ``parent`` alive. The :py:class:`equistore.data.ExternalCpuArray` class
+    should provide the right behavior for data living in CPU memory, and can
+    serve as an example for more advanced custom arrays.
```

Comment:
Which German wrote this.

But in all seriousness, is the spelling as `klass` intentional? Why?

Reply:
Ich bin der Deutsche!

I used `klass` because `class` is a keyword and can not be used as a parameter name. Typical alternative are `cls`, `_class`, but I kinda like the german version

Reply:
While I appreciate good deviant behavior, it might cause some confusion not to use the typical nomenclature. Up to you!